Control/Monad/Logger.hs

```diff
@@ -680,11 +681,11 @@ defaultLogStrWithoutLoc loc src level msg =
 -- | Run a block using a @MonadLogger@ instance which appends to the specified file.
 --
 -- @since 0.3.22
-runFileLoggingT :: MonadBaseControl IO m => FilePath -> LoggingT m a -> m a
+runFileLoggingT :: MonadUnliftIO m => FilePath -> LoggingT m a -> m a
```

Comment:
There is not!

Reply:
OK. If you don't mind bouncing ideas around, I'd appreciate the feedback. My general approach with the move towards `MonadUnliftIO` and `ReaderT` has been that it generally simplifies things to put the constraints on the environment instead of the monad, and to use `Has`-style typeclasses. This is the approach I took for logging in `rio`:

https://www.stackage.org/haddock/lts-12.2/rio-0.1.4.0/RIO.html#g:1

I'm considering moving monad-logger over to using `rio` under the surface, and adapting the current API as needed to make that as painless as possible. Any thoughts?

Reply:
I'm happy to provide feedback but I'm just about to exercise, then crash (UTC -4 here). I'll get back to you tomorrow!

Btw, this is exactly the kind of discussion I was imagining using a completely new forum for, as none of Twitter/Reddit/IRC seem to suffice. It's at github.com/haskell-api-discussions and has no users at the moment ;)

Reply:
Would you mind if I make a ticket in this repo to discuss so others can chime in?

Reply:
No objection.